app/controllers/referee_interface/reference_controller.rb

```diff
@@ -38,7 +38,13 @@ def confirmation; end
     def confirm_consent
       consent_to_be_contacted = params.dig(:application_reference, :consent_to_be_contacted)
 
-      reference.update!(consent_to_be_contacted: consent_to_be_contacted)
+      reference.update!(consent_to_be_contacted: consent_to_be_contacted, refuse_to_give_feedback: false)
+
+      render :finish
+    end
+
+    def refuse_to_give_feedback
```

Comment:
`GET` requests should never change something, I think we should have a page here that allows the referee to confirm that they don't want to give feedback. Do we have designs for the feature?

Reply:
You're right, but no design atm. In the first implementation, I assumed that it redirect to the thank you page.

Yesterday I've put a comment to ask for more info about this on trello: https://trello.com/c/3kuEKaQH/666-update-email-content-for-referee-request